package.json

```diff
@@ -26,8 +26,8 @@
     "xstate": "^4.7.1"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "start": "./scripts/watch.js",
+    "build": "INLINE_RUNTIME_CHUNK=false react-scripts build",
```

Comment:
We don't generate inline chunks in the final `index.html` file because it goes against Chrome's CSP. [More details here](https://mmazzarolo.com/blog/2019-10-19-browser-extension-development/), on step 3.